timescaledb/how-to-guides/install-timescaledb/installation-apt-debian.md

```diff
@@ -19,25 +19,19 @@ of `apt`, we recommend installing from source.  Otherwise, please be
 sure to remove non-`apt` installations before using this method.
 </highlight>
 
-**If you don't already have PostgreSQL installed**, add PostgreSQL's third
-party repository to get the latest PostgreSQL packages:
+Add the PostgreSQL third party repository to get the latest PostgreSQL packages:
 ```bash
-# `lsb_release -c -s` should return the correct codename of your OS
-echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -c -s)-pgdg main" | sudo tee /etc/apt/sources.list.d/pgdg.list
-wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
-sudo apt-get update
+sudo apt install postgresql-common
+sudo sh /usr/share/postgresql-common/pgdg/apt.postgresql.org.sh
```

Comment:
I guess by using this script, it becomes "not our problem", but this script still uses `apt-key` to install the postgres keyring :)

Reply:
who owns the script? pg?

Reply:
Unclear. It is mentioned on https://wiki.postgresql.org/wiki/Apt where it says Debian and Ubuntu include the script. But the script itself is hosted on a debian.org host, so perhaps it is owned by the Debian postgres maintainers.

Reply:
Definitely in the 'not our problem' basket, then, I think 😉